site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -426,18 +426,14 @@ Now, let's create our user. Here, we can specify optional user roles in our appl
 user_registration_request = {
     'registration': {
         'applicationId': application_id,
-        'roles': [
-            'user',
-            'community_helper'
-        ]
+        'roles': []
```

Comment:
this will break if roles are not already defined in FusionAuth

Reply:
Should we create the roles in the application above rather than remove them?

Reply:
I added a callout stating the roles must be defined in FusionAuth to be assigned to a user

Reply:
I'd change the callout to say where you define the fusionauth roles (in an application) but other than that, looks great!

Reply:
fixed in latest commit